config/kubernetes/deploy-kbs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
nit: `/bin/bash` isn't available on all systems,